azavea/raster-foundry
# Add lab map component and use in lab2

## Overview

Add map component with ability to resize to lab
All map features that were previously available should work now, including split preview

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Testing Instructions

* Verify that the dragging works as expected (no weird highlighting, etc)
* Verify that you can maximize the diagram or the map by dragging the slider all the way to one side
* Verify that the split preview comparison works
* Verify that closing the preview then re-opening it preserves the last split position

Closes #2163
